docs/advanced.rst

```diff
@@ -562,3 +562,79 @@ or the record invocation: ``app="SomeAppName"``.
         opinion = models.CharField(max_length=2000)
 
     register(Opinion, app="SomeAppName")
+
+Custom History Model Name
+-------------------------
+
+By default the history model names are the base model name prefixed with "Historical".
+So given a model named ``Poll``, the resultant history class will be named ``HistoricalPoll``.
+If the app name of the ``Poll`` model is `election` then the history table name would be ``election_historicalpoll``.
+
+This feature allows a different and/or shorter model name and would even allow model names to be in the standard Django form
+
+    [appName]_[modelName]
```

Comment:
Is this the standard Django form? Can't think of when models are referenced using their app name unless using the `get_model` function

Reply:
Yes was not well written - Have reworded it